components/provisioning/org.wso2.carbon.identity.provisioning/src/main/java/org/wso2/carbon/identity/provisioning/dao/CacheBackedProvisioningMgtDAO.java

```diff
@@ -35,6 +35,11 @@ public class CacheBackedProvisioningMgtDAO {
 
     private ProvisioningEntityCache provisioningEntityCache = null;
 
+    public CacheBackedProvisioningMgtDAO() {
+        this.provisioningMgtDAO = null;
+        this.provisioningEntityCache = null;
+    }
+
```

Comment:
what is the reason for this constructor with null initialization ?

Reply:
removed